tests/test-cases/Group1-Docker-Commands/1-19-Docker-Volume-Create.robot

```diff
@@ -8,86 +8,62 @@ Suite Teardown  Cleanup VIC Appliance On Test Server
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} volume create -d fakeDriver --name=test2
+    Should Be Equal As Integers  ${rc}  1
+    Should Contain  ${output}  Error looking up volume plugin fakeDriver: plugin not found
     
 Docker volume create with bad volumestore
-    ${status}=  Get State Of Github Issue  1561
-    Run Keyword If  '${status}' == 'closed'  Fail  Test 1-19-Docker-Volume-Create.robot needs to be updated now that Issue #1561 has been resolved
-    Log  Issue \#1561 is blocking implementation  WARN
-    #${rc}  ${output}=  Run And Return Rc And Output  docker ${params} volume create --name=test3 --opt VolumeStore=fakeStore
-    #Should Be Equal As Integers  ${rc}  1
-    #Should Contain  ${output}  Error looking up volume store fakeStore: datastore not found
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} volume create --name=test3 --opt VolumeStore=fakeStore
+    Should Be Equal As Integers  ${rc}  1
+    Should Contain  ${output}  No volume store named (fakeStore) exists
 
 Docker volume create with specific capacity
-    ${status}=  Get State Of Github Issue  1565
-    Run Keyword If  '${status}' == 'closed'  Fail  Test 1-19-Docker-Volume-Create.robot needs to be updated now that Issue #1565 has been resolved
-    Log  Issue \#1565 is blocking implementation  WARN
-    #${rc}  ${output}=  Run And Return Rc And Output  docker ${params} volume create --name=test4 --opt Capacity=100
-    #Should Be Equal As Integers  ${rc}  0
-    #Should Be Equal As Strings  ${output}  test4
-    #${rc}  ${output}=  Run And Return Rc And Output  govc datastore.ls -json=true test/VIC/volumes/test4
-    #Should Be Equal As Integers  ${rc}  0
-    #Should Contain  ${output}  "FileSize":100
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} volume create --name=test4 --opt Capacity=100000
+    Should Be Equal As Integers  ${rc}  0
+    Should Be Equal As Strings  ${output}  test4
+    ${disk-size}=  Run  docker ${params} logs $(docker ${params} start $(docker ${params} create -v ${output}:/mydata busybox /bin/df -Ph) && sleep 10) | grep by-label | awk '{print $2}' 
+    Should Be Equal As Strings  ${disk-size}  96.0G
     
 Docker volume create with zero capacity
-    ${status}=  Get State Of Github Issue  1562
-    Run Keyword If  '${status}' == 'closed'  Fail  Test 1-19-Docker-Volume-Create.robot needs to be updated now that Issue #1562 has been resolved
-    Log  Issue \#1562 is blocking implementation  WARN
-    #${rc}  ${output}=  Run And Return Rc And Output  docker ${params} volume create --name=test5 --opt Capacity=0
-    #Should Be Equal As Integers  ${rc}  1
-    #Should Contain  ${output}  Error
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} volume create --name=test5 --opt Capacity=0
+    Should Be Equal As Integers  ${rc}  1
+    Should Contain  ${output}  Error
```

Comment:
I will take a look at the most recent logs. It should be reasonable. Should be an `invalid size` type error